.github/workflows/publish-v2.yml

```diff
@@ -86,14 +86,35 @@ jobs:
           environment-id: "${{ env.PACKAGE_VERSION }}"
           environment-display-name: "${{ env.PACKAGE_VERSION }}"
           environment-type: 'production'
+      - name: Debug Publish to WordPress.org SVN
+        if: env.CHANNEL != 'cloud' && github.event.inputs.pre_release == 'true'
+        env:
+          PLUGIN_VERSION: ${{ env.RELEASE_NAME }}
+          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          CHANNEL: ${{ env.CHANNEL }}
+          PACKAGE_VERSION : ${{ env.PACKAGE_VERSION }}
+        run: |
+          unzip -q /tmp/${{ github.event.repository.name }}-${{ env.RELEASE_FILENAME }}.zip -d $GITHUB_WORKSPACE
+          bash "${GITHUB_WORKSPACE}/.github/scripts/validate-build-files.sh"
```

Comment:
Don't we want to run the validation also on regular releases?

Reply:
As you can see in the link, we do validate also on regular releases: 

https://github.com/maork-elementor/elementor/blob/368839e3caaf30594d7922b08855b0be0a600b48/.github/workflows/publish-v2.yml#L117

Reply:
Done